.vscode/launch.json

```diff
@@ -16,7 +16,7 @@
             "outFiles": [
                 "${workspaceFolder}/dist/**/*.js"
             ],
-            "preLaunchTask": "npm: watch",
+            "preLaunchTask": "npm: test-compile",
```

Comment:
Not sure about this as part of this PR - if we want to do this let's break it out for further discussion.  We have direct experience of webpack introducing errors and I'd greatly prefer to pick those up at this stage of the cycle.

Reply:
I forgot to have changed it, i had some issue to debug it. Reverted :+1: